examples/dialtone.py

```diff
@@ -17,13 +17,6 @@
 import psycopg2
 from psycopg2.extensions import adapt, register_adapter
 
-try:
-    sorted()
```

Comment:
Ha ha, this is also broken by the way :D `sorted()` always gives an error: missing argument :P It should have been `sorted` and catching `ImportError`...

Ok, these directories are largely archaeology anyway: I guess we are leaving them here for sentimental reasons...